src/App.jsx

```diff
@@ -1,13 +1,16 @@
 import "./App.css"
+import "./components/common/styles.scss"
 import { Route, Routes } from "react-router-dom"
-import HomePage from "./components/HomePage/HomePage"
-import LoginScreen from "./components/LoginScreen/LoginScreen"
-import SettingScreen from "./components/SettingScreen/SettingScreen"
+import HomePage from "./components/pages/HomePage/Index"
+import LoginScreen from "./components/pages/LoginScreen/LoginScreen"
+import SettingScreen from "./components/pages/SettingScreen/SettingScreen"
+import SignUpScreen from "./components/pages/SignUpScreen/SignUpScreen"
 function App() {
 	return (
 		<>
 			<Routes>
 				<Route path="/" element={<LoginScreen />} />
+				<Route path="sign-up" element={<SignUpScreen />} />
```

Comment:
Insert "/" before "sign-up"

Reply:
Bonus: Move to constant
Example: 
const Routers = {
 path: '/',
 component: <Component />
 ...
}

Reply:
routes has been transfer to main.jsx and re-setup as react-router-doom.v6 suggestion